src/app/backend/resource/service/list_test.go

```diff
@@ -15,6 +15,8 @@
 package service
 
 import (
+	"github.com/kubernetes/dashboard/src/app/backend/resource/endpoint"
```

Comment:
These two imports should be after L25 ("github.com/kubernetes/dashboard/src/app/backend/resource/dataselect")